Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionContextPrivate.h

```diff
@@ -35,6 +35,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, getter=_inTestingMode, setter=_setTestingMode:) BOOL _testingMode;
 
+/*! @abstract The extension background view used for the extension, or `nil` if the extension does not have background content or it is currently unloaded. */
```

Comment:
Are there certain files where we should use "`nil`" vs "\c nil"?

Reply:
The backticks work in this, just not `@param`.